README.md

```diff
@@ -1,24 +1,56 @@
+# ![logo](.github/logo-small.png) Alpha Core
+
+
+# Enjoy or want to support the project?
+
 [![ko-fi](https://www.ko-fi.com/img/githubbutton_sm.svg)](https://ko-fi.com/R6R21LO82)
 
-# alpha-core
-0.5.3 experimental emulator written in Python.
 
-## Installation
-You need Python 3.9+ and a MariaDB server. For project requirements, install them with `pip3 install -r requirements.txt`.
+## The Alpha Project - Alpha-core
+
+Based on `0.5.3` Alpha version of World of Warcraft, `alpha-core` is an experimental emulator written in Python.
```

Comment:
Maybe it can be changed to something along those lines:

> `alpha-core` is an experimental emulator written in Python for version `0.5.3` of the Friends & Family Alpha of World of Warcraft.